src/components/Modals/PublishWizardCollectionModal/CongratulationsStep/CongratulationsStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Button, Column, Modal, Row } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Collection } from 'modules/collection/types'
+import './CongratulationsStep.css'
+
+export const CongratulationsStep: React.FC<{ collection: Collection }> = props => {
+  const { collection } = props
+
+  return (
+    <>
```

Comment:
nitpick: I believe we don't need the fragment here, right? 
```suggestion
    <Modal.Content className="CongratulationsStep">
```